docs/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,219 @@
+    ```
+    If the `query` argument isn't supplied, a default list of arguments will be used.
+    See the [API reference](https://github.com/Shopify/js-buy-sdk/blob/v1.0alpha/docs/API_REFERENCE.md#Client.Queries) for more details.
+
+3.  `fetchCollection()` and `fetchAllCollections()` no longer return the products in the collections by default.
+    To fetch products on the collections, use `fetchCollectionWithProducts()` and `fetchAllCollectionsWithProducts()`,
+    or specify products in the query function parameter.
+    ```js
+    const collectionId = 'Z2lkOi8vc2hvcGlmeS9Db2xsZWN0aW9uLzI1NzY5NzczMQ=='
+    
+    // Use the built-in function
+    client.fetchCollectionWithProducts(collectionId).then((collection) => {
+      console.log(collection); // Collection with all default fields and products with all default fields.
+    });
+
+    // or specify a custom query
+    const query = collectionNodeQuery(['title', ['products', productConnectionQuery(['title'])]]);
+
+    client.fetchCollection(collectionId, query).then((collection) => {
+      console.log(collection); // Collection with only title and products with only title
+    });
+    ```
+    Note that `fetchCollectionWithProducts()` and `fetchCollectionsWithProducts()` do not take a `query` argument.
+
+4.  `fetchQueryProducts()` and `fetchQueryCollections()` query different fields and take an optional `query` argument.
+    See the [API reference](https://github.com/Shopify/js-buy-sdk/blob/v1.0alpha/docs/API_REFERENCE.md) for more details.
+
+### Carts/Checkouts
+
+Carts are replaced with checkouts. Like the fetch functions, all checkout functions take an optional `query` argument that specifies fields to return on the checkout.
+
+`fetchRecentCart()`, `updateModel()`, and `clearLineItems()` have been removed. `checkoutUrl` has been renamed to `webUrl`.
+
+#### Creating a Checkout
+
+To create a checkout, use `createCheckout()`.
+
+**v0.7:**
+```js
+client.createCart().then((cart) => {
+  console.log(cart); // Empty cart
+});
+```
+
+**v1:**
+```js
+client.createCheckout().then((checkout) => {
+  console.log(checkout); // Empty checkout
+});
+```
+The checkout can also be initialized with fields like line items and a shipping address. See the [API reference](https://github.com/Shopify/js-buy-sdk/blob/v1.0alpha/docs/API_REFERENCE.md#Client+createCheckout) for more details. 
+
+#### Fetching a Checkout
+
+To fetch a checkout, use `fetchCheckout()`.
+
+**v0.7:**
+```js
+client.fetchRecentCart().then((cart) => {
+  console.log(cart); // Most recently created cart
+});
+
+// or
+client.fetchCart('shopify-buy.1459804699118.2').then(cart => {
+  console.log(cart); // The retrieved cart
+});
+```
+
+**v1:**
+```js
+const checkoutId = 'Z2lkOi8vc2hvcGlmeS9Qcm9kdWN0SW1hZ2UvMTgyMTc3ODc1OTI=';
+
+client.fetchCheckout(checkoutId).then((checkout) => {
+  console.log(checkout); // The retrieved checkout
+});
+```
```

Comment:
Show an example of `fetchRecentCart` and then a v1 version where they use `localStorage.setItem` and `localStorage.getItem` to save and fetch and ID.